api/src/repositories/technique-attribute-repository.ts

```diff
@@ -0,0 +1,529 @@
+import { z } from 'zod';
+import { getKnex } from '../database/db';
+import { ApiExecuteSQLError } from '../errors/api-error';
+import { getLogger } from '../utils/logger';
+import { BaseRepository } from './base-repository';
+
+const defaultLog = getLogger('repositories/technique-attribute-repository');
+
+export interface IQuantitativeAttributePostData {
+  method_technique_attribute_quantitative_id?: number;
+  method_lookup_attribute_quantitative_id: string;
+  value: number;
+}
+
+export interface IQualitativeAttributePostData {
+  method_technique_attribute_qualitative_id?: number;
+  method_lookup_attribute_qualitative_option_id: string;
+  method_lookup_attribute_qualitative_id: string;
+}
+
+// export interface IQuantitativeAttributeRecord {
```

Comment:
can we drop this dead code?💀

Reply:
Yes! I noticed there was some redundancy before, but must have forgotten to remove the old interfaces. Fixed